jbrowse/src/client/JBrowse/Browser/plugins/ExtendedVariantPlugin/ExtendedVariantWidget/index.js

```diff
@@ -10,14 +10,15 @@ export default jbrowse => {
       id: ElementId,
       type: types.literal('ExtendedVariantWidget'),
       featureData: types.frozen({}),
+      widgetDisplayInfo: types.frozen([])
```

Comment:
For the sake of consistency, i'd rename this to match whatever we end up calling the property under track.metadata

Reply:
Renamed.